### ISSUE TITLE
Deprecation warnings with `Dart Sass 1.33.0`

### ISSUE THREAD

**Issue by boris-petrov** [twbs/bootstrap-sass#1217]
`Dart Sass` [deprecated](https://sass-lang.com/documentation/breaking-changes/slash-div) the division syntax in version 1.33.0. Bootstrap [uses](https://github.com/twbs/bootstrap-sass/blob/v3.4.1/assets/stylesheets/bootstrap/mixins/_grid-framework.scss#L41) this a lot and there are hundreds of warnings in my console. It would be nice to fix those.

**Comment by Koozza** [comment]
Sadly same issue here. Would love to see this fixed.

**Comment by realtebo** [comment]
We will end to use https://www.npmjs.com/package/patch-package

Sad but ... we need to continue use this package and do not wait so long because of 3 minutes of deprecation warnings!

**Comment by Zianor** [comment]
Any plans to merge the PR to fix the issue?

**Comment by japick-joules** [comment]
Same issue but for version 2.0.0

`Deprecation Warning: Using / for division outside of calc() is deprecated and will be removed in Dart Sass 2.0.0.`

**Comment by OakBehringer** [comment]
Considering this library hasn't had a single commit made to it in years, I'd say we're SOL

**Comment by jordanade** [comment]
Now I can't compile at all:
```Module build failed (from ./node_modules/mini-css-extract-plugin/dist/loader.js):
ModuleBuildError: Module build failed (from ./node_modules/sass-loader/dist/cjs.js):
SassError: Missing argument $number2.
  ┌──> node_modules/bootstrap-sass/assets/stylesheets/bootstrap/_variables.scss
338│ $grid-float-breakpoint-max: math.div($grid-float-breakpoint - 1) !default;
  │                             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ invocation
  ╵
```

**Comment by AprilArcus** [comment]
> @glebm Now I can't compile at all:
> 
> ```
> ModuleBuildError: Module build failed (from ./node_modules/sass-loader/dist/cjs.js):
> SassError: Missing argument $number2.
>   ┌──> node_modules/bootstrap-sass/assets/stylesheets/bootstrap/_variables.scss
> 338│ $grid-float-breakpoint-max: math.div($grid-float-breakpoint - 1) !default;
>   │                             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ invocation
>   ╵
> ```

fixed in 3.4.3; sorry about that.
